src/shared/components/proteinChainPanel/column/OrganismColumnFormatter.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import {PdbHeader} from "../../../api/generated/PdbAnnotationAPI";
+import * as _ from "lodash";
+import PdbHeaderCache from "../../../cache/PdbHeaderCache";
+import {IPdbChain} from "../../../model/Pdb";
+
+export default class OrganismColumnFormatter {
+    public static getOrganism(pdbHeader:PdbHeader, chainId:string):string {
```

Comment:
Maybe move this function into `PdbUtils` for consistency? We already have a similar function there: https://github.com/cBioPortal/cbioportal-frontend/blob/integration/src/shared/lib/PdbUtils.ts#L11